apps/bridge-dapp/src/components/InteractiveFeedbackView/DefaultErrorView.tsx

```diff
@@ -0,0 +1,113 @@
+import { FeedbackEntry } from '@webb-tools/dapp-types';
```

Comment:
I created this component on the bridge because we still need a design for this, another reason is that I want to make sure this component does not depend on other libs before moving it to the `webb-ui-kit`.

Reply:
But good catch here 👍

Reply:
That would extend to the other components included in this PR. We can do this in a subsequent PR(s).